### DISCUSSION TITLE
Using With-input plugin in conjunction with Prisma plugin

### DISCUSSION THREAD

**Discussion by 5t111111** [hayes/pothos#600] [Q&A] [answered]
Prisma plugin works amazingly well and With-input plugin looks very handy for co-located schema definition, but it seems With-input plugin cannot be used with Prisma field.

I think it is very convenient  if you can use With-input plugin along with Prisma plugin. Is there any way to use both in conjunction?

**Comment by hayes** [comment, marked as answer]
Yup, this is something on my todo list: https://github.com/hayes/pothos/issues/408, just haven't gotten around to it yet

**Reply by 5t111111** [reply]
Oops I didn't find that issue. Thanks, I'm eagerly awaiting it!

**Reply by hayes** [reply]
new version with this has been released now
